toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -305,8 +305,13 @@ func findSPECFiles(specsDir string, packList []string) (specFiles []string, err
 				return
 			}
 			if len(specFile) != 1 {
-				err = fmt.Errorf("unexpected number of matches (%d) for spec file (%s)", len(specFile), specName)
-				return
+				if( specName == "msopenjdk-11" ) {
+					logger.Log.Infof("Ignoring missing match for (%s)", specName)
+					continue
+				} else {
```

Comment:
Here I would also try to be more flexible with the version and logging, since it might be confusing for the user:

```suggestion
				if (strings.HasPrefix(specName, "msopenjdk-11" ) {
					logger.Log.Debug("Ignoring missing match for the 'msopenjdk' package, which is externally-provided and thus doesn't have a local spec.")
					continue
				} else {
```

Reply:
sounds good, will update